tests/nullish_coalescing/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -28,7 +33,12 @@ foo ? bar ?? foo : baz;
 
 foo ?? (bar ?? baz);
 
-foo ?? baz || baz;
+(foo ?? baz) || baz;
+
+// Note: this will trigger a syntax error once the parsers have been
+// updated to the latest specification. If you are doing the upgrade,
+// please remove the following line (and this comment).
```

Comment:
Instead of deleting right now, I added a comment for the person that does the upgrade later.